validation/src/main/java/io/micronaut/validation/validator/constraints/DefaultConstraintValidators.java

```diff
@@ -45,13 +45,7 @@
 import java.time.chrono.MinguoDate;
 import java.time.chrono.ThaiBuddhistDate;
 import java.time.temporal.TemporalAccessor;
-import java.util.Arrays;
-import java.util.Collection;
-import java.util.Collections;
-import java.util.HashMap;
-import java.util.Map;
-import java.util.Objects;
-import java.util.Optional;
+import java.util.*;
```

Comment:
Could you revert this change and use explicit imports

Reply:
Done.